flang/module/iso_c_binding.f90

```diff
@@ -22,15 +22,32 @@ module iso_c_binding
     c_sizeof => sizeof, &
     operator(==), operator(/=)
 
+  implicit none
+
+  ! Set PRIVATE by default to explicitly only export what is meant
+  ! to be exported by this MODULE.
+  private
+
+  public :: c_associated, &
```

Comment:
If you're going to put each of these on its own line, please don't have a line with two.  Or just cram them all on to a few lines to save vertical space.

Reply:
Alright, that's an easy one.  In my defense I have copied the USE statement and kept the splitting.

Reply:
Fixed.